.github/workflows/ci_check_pr_title.yml

```diff
@@ -0,0 +1,52 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: InLong Check Pull Request Title
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+
+jobs:
+  check:
+    name: Check pull request title
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # The correct title should start with [WIP] or [INLONG-xxx],
+      # where xxx is the issue number, e.g. [INLONG-123].
+      - name: Check title
+        if: ${{ github.event_name == 'pull_request_target' }}
+        run: |
+          if [[ ! "$PR_TITLE" =~ ^\[(WIP|INLONG-[0-9]+)\].*$ ]]; then
+            echo "This pull request title is not valid."
+            echo "The correct title should start with [WIP] or [INLONG-xxx], where xxx is the issue number, e.g. [INLONG-123]."
+          
+            # add a job summary
+            echo "## This pull request title is not valid! :bug:" >> $GITHUB_STEP_SUMMARY
+            echo "The correct title should start with \`[WIP]\` or \`[INLONG-xxx]\`, where **xxx** is the issue number, e.g. \`[INLONG-123]\`." >> $GITHUB_STEP_SUMMARY
```

Comment:
If the title is not valid, the job summary will be added.
![image](https://user-images.githubusercontent.com/49934421/173216172-9253e02e-b9a4-4075-b4e3-6a91a6d9a6b9.png)

Reply:
See https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary